@here/harp-examples/src/getting-started_globe-projection.ts

```diff
@@ -53,7 +59,12 @@ export namespace GlobeExample {
             copyrightInfo: copyrights
         });
 
+        const polarDataSource = new PolarTileDataSource({
```

Comment:
I think it would be better to handle this internally and not put the burden on the user to add this data source

Reply:
Will think how it can be done automatically.